rofeeah_ayeni/alpha_challenges/conditionals.md

```diff
@@ -6,16 +6,34 @@ Chartreuse is my favorite color. Magenta is my second favorite. Periwinkle is my
 * In Ruby, write a conditional that will check a variable set to a color and tell me how I feel about it.
 
 ```ruby
-color = "Magenta"
+color = ""
 
-# write your code here
+case color
+when "Chartreuse"
+  puts "That's my favorite color"
+when "Magenta"
+  puts "That's my second favorite color"
+when "Periwinkle"
+  puts "This is my third favorite color"
+else
+  puts "Yuck! I hate this color."
+end
 ```
 
 Now, turn your color conditional into a method called `color_checker` that takes any color as an argument.
 
 ```ruby
 def color_checker(color)
-  # write your code here
+case color
+when "Chartreuse"
+  puts "That's my favorite color"
+when "Magenta"
+  puts "That's my second favorite color"
+when "Periwinkle"
+  puts "This is my third favorite color"
+else
+  puts "Yuck! I hate this color."
+end
```

Comment:
Nice work with the `case` statement, but -- it's nested within the method, so you know what that means...tabs! 😄

Reply:
how did I forget the most important thing?!! LOL